hydroshare/irods/ruleGenerateBagIt_HS.r

```diff
@@ -81,12 +81,16 @@ generateBagIt {
   msiFreeBuffer("stdout");
 
   ### - writes tagmanifest file to BAGITDATA/tagmanifest-md5.txt
-  msiDataObjChksum("*BAGITDATA" ++ "/bagit.txt", "forceChksum", *CHKSUM);
+  msiDataObjChksum("*BAGITDATA" ++ "/bagit.txt", "forceChksum=", *CHKSUM);
```

Comment:
both forms of parameter passing work, but want this call to be consistent with the ```msiDataObjChecksum``` call above it so that they can all be passed in the same form of ```forceChksum=``` with an equal sign.